typescript/react/security/audit/react-styled-components-injection.jsx

```diff
@@ -2,28 +2,22 @@ import styled from "styled-components";
 
 function Vulnerable1(userInput) {
 // ruleid: react-styled-components-injection
```

Comment:
I've also come across cases like this. I wonder if there is a better way to do this, instead of making the code a single line. Should we have smarter test syntax?

Reply:
Oh actually it works fine with comments inside the curly braces! Fixed in dd312e9fd303100c1c7138fbf2a9b1cee25c9cb3